--- src/libmatheval-1.1.11/Makefile.am	2013-08-14 22:22:08.000000000 +0200
+++ ../patches/Makefile.am	2017-07-24 16:24:19.960705300 +0200
@@ -27,7 +27,7 @@
 MAINTAINERCLEANFILES = Makefile.in aclocal.m4 configure config.h.in	\
 config.log config.status stamp-h.in $(AUX_DIST)
 
-SUBDIRS = doc lib tests
+SUBDIRS = doc lib
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libmatheval.pc
@@ -35,7 +35,7 @@
 .PHONY: beauty dist-hook
 
 beauty:
-	-for dir in lib tests; do cd $$dir; $(MAKE) $@; cd ..; done
+	-for dir in lib; do cd $$dir; $(MAKE) $@; cd ..; done
 
 dist-hook:
 	for file in $(AUX_DIST); do \
